security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainResourceDefinition.java

```diff
@@ -63,9 +64,12 @@
  */
 class SecurityDomainResourceDefinition extends SimpleResourceDefinition {
 
+    static final String INFINISPAN_CACHE_TYPE = "infinispan";
+    static final RuntimeCapability<Void> INFINISPAN = RuntimeCapability.Builder.of("org.wildfly.security.domain.infinispan", true).build();
```

Comment:
I'll poke at creating a general capability for legacy security domains today while I wait for my kid's choir practice. ;) We need to do it at some point regardless in order to complete cap/req coverage, and the bit of poking I did last night after my comment above makes me think it *might* be simple.